modules/op-release-notes-1-9.adoc

```diff
@@ -0,0 +1,240 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+:_content-type: REFERENCE
+[id="op-release-notes-1-9_{context}"]
+= Release notes for {pipelines-title} General Availability 1.9
+
+With this update, {pipelines-title} General Availability (GA) 1.9 is available on {product-title} 4.11 and later versions.
+
+[id="new-features-1-9_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.9.
+
+[id="pipelines-new-features-1-9_{context}"]
+=== Pipelines
+
+* With this update, you can specify pipelines parameters and results in arrays and object dictionary forms.
+
+* This update provides support for Container Storage Interface (CSI) and projected volume for your workspace.
+
+* This update supports the matrix feature to organize tasks in a pipeline. For example, you can define a single task in your pipeline which executes multiple times based on the matrix parameters. This update is part of an alpha preview feature.
```

Comment:
"alpha preview"

Is this the correct terminology? developer preview maybe?

Reply:
@Preeticp @concaf , need your input here. can we replace alpha preview with developer preview?

Reply:
@rupalibehera @piyush-garg @ppitonak can you weigh in on this, please? I would say yes.

Reply:
Changed to developer preview

Reply:
If I understand it correctly, customers need to set `enable-api-fields: alpha` in TektonConfig which makes their installation unsupportable. I'm not sure if we should even mention it

Reply:
@vdemeester could you please provide your input here? do we need to remove this point altogether?

Reply:
ah you are right @ppitonak, this is still in alpha, so, we could not mention it indeed.

Reply:
Removed this point @ppitonak @vdemeester